cdc/sink/mysql.go

```diff
@@ -271,6 +271,11 @@ type sinkParams struct {
 	batchReplaceSize    int
 }
 
+func (s *sinkParams) Clone() *sinkParams {
```

Comment:
I think the easiest way to clone is using value instead of pointer.

Reply:
yes, but this is a more uniform implementation.